--- nemo-5.8.2/libnemo-private/nemo-search-engine-tracker.c.orig	2023-06-14 09:26:04.000000000 +0200
+++ nemo-5.8.2/libnemo-private/nemo-search-engine-tracker.c	2023-06-16 17:29:02.298346219 +0200
@@ -129,7 +129,7 @@ cursor_callback (GObject      *object,
 
 	/* We iterate result by result, not n at a time. */
 
-    fsr = file_search_result_new (g_strdup (tracker_sparql_cursor_get_string (cursor, 0, NULL)));
+    fsr = file_search_result_new (g_strdup (tracker_sparql_cursor_get_string (cursor, 0, NULL)), NULL);
     hits = g_list_append (NULL, fsr);
     nemo_search_engine_hits_added (NEMO_SEARCH_ENGINE (tracker), hits);
     g_list_free_full (hits, (GDestroyNotify) file_search_result_free);
